packages/swapper/src/swappers/PortalsSwapper/utils/fetchPortalsTradeOrder.ts

```diff
@@ -50,6 +52,25 @@ type PortalsTradeOrderResponse = {
     gasLimit: string
   }
 }
+
+type PortalsTradeOrderEstimateResponse = {
+  outputAmount: string
+  minOutputAmount: string
+  outputToken: string
+  outputTokenDecimals: number
+  context: {
+    slippageTolerancePercentage: number
+    inputAmount: string
+    inputAmountUsd: number
+    inputToken: string
+    outputToken: string
+    outputAmount: string
+    outputAmountUsd: number
+    minOutputAmountUsd: number
+    sender: string
```

Comment:
This should be optional according to the swagger spec

<img width="813" alt="image" src="https://github.com/user-attachments/assets/caa4b053-a016-4abe-bf33-2ed7b4664821">

Reply:
https://github.com/shapeshift/web/pull/7672/commits/102812593a73ebd529202b70c88a65b76f9f0f5c 👌